src/main/java/ch/njol/skript/expressions/ExprIndices.java

```diff
@@ -76,20 +99,33 @@ public boolean isSingle() {
 	
 	@Override
 	public String toString(@Nullable Event e, boolean debug) {
+		StringBuilder builder = new StringBuilder("all indices of ");
```

Comment:
Any reason you are using StringBuilder explicitly here? Java compiler can generate StringBuilder code on the fly and it should (generally) only be used explicitly on loops as far as my knowledge about Javac.

Reply:
Mostly for readability's sake, when you think of StringBuilder you think of concatenating stuff to a string, but it's not as obvious if you're just appending to a local variable :D

I can change it if you think it'd be better as concatenation :)

Reply:
Did that as well, should work (I wrote it on my phone, but I can dream lol)